fix_tests.py

```
@@ -0,0 +1,22 @@
+import re
+import os
+
+def process_file(filepath):
+    with open(filepath, 'r') as f:
```

Ruff (0.15.13)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)